### ISSUE TITLE
Obvious to balance here

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#1003]
1C2DD8E931B07E9836C0DB567063
![image](https://github.com/user-attachments/assets/079c8391-6dc3-4c88-96eb-345c4fcfd1e6)

East last PASS is wrong

![image](https://github.com/user-attachments/assets/a83a9200-9e3e-4571-aae0-63d7487b63c6)

As we are now in the balancing situation there is no reason to jump to 3H, just bid 2H 

Now over 2D compete in 3H, and here is another error as 3H show 0-4 and instead should be 5+ and 5-card hearts
![image](https://github.com/user-attachments/assets/ee56cdfe-998e-451a-a730-1b792b0311c2)

**Comment by EdwardPiwowar** [comment]
Next BBA
![image](https://github.com/user-attachments/assets/4757a26d-015f-483e-aae1-cad5a7c6cecb)
3H is still weak.

**Comment by ThorvaldAagaard** [comment]
In 8719 North now jump to 3D
![image](https://github.com/user-attachments/assets/6b9a6c36-35ac-48dd-9477-9ee121a562ee)

**Comment by EdwardPiwowar** [comment]
3D if 1D means 6+ HCP
2D if 1D means 5+ HCP

**Comment by ThorvaldAagaard** [comment]
I do not agree, it is a 2D response. On 3D south will often bid 3N on 8-9 balanced, and then we have created a real poblem, as can't gfet out in 4D, and 5D and 3N will probably have no play

Generally it is a bad habbit to try to compensate for partner answering below the range.

So 1D is 6+ HCP, but there might be a few hands, that responds 1D on 4-5 HCP
